Comparison shopping engine/Forms/SearchForm.cs

```diff
@@ -100,9 +100,7 @@ private void ToggleSourcePanel(object sender, EventArgs e)
 
         private void SearchForm_Load(object sender, EventArgs e)
```

Comment:
Jeigu metodas tuščias tai ar nebūtų galima jo visai ištrinti?

Reply:
cia form load metodas, be jo ismes errora

Reply:
Tai kaip aš suprantu errora meta dėl to, kad šis metodas yra priskirtas tai formai, jeigu nebūtų tai ir nemestų